apps/nextjs-website/src/components/organisms/ChatbotLayout/ChatbotLayout.tsx

```diff
@@ -53,7 +53,6 @@ const ChatbotLayout = ({
         aria-describedby={id}
         isChatOpen={open}
         onOpenChat={handleClick}
-        hasNewMessages={false}
```

Comment:
Is correct to remove `hasNewMessages` :thinking: ?
Because i don't know if it is outdated or not but i still see in figma a version of this button with notification:
![Screenshot from 2024-08-06 14-51-50](https://github.com/user-attachments/assets/5ef51278-bfaa-44c5-bd81-952c2fb59e61)

Reply:
Yes, the feature has been put on hold for now